modules/storage-api/src/testFixtures/java/org/apache/ignite/internal/storage/index/impl/TestSortedIndexStorage.java

```diff
@@ -139,6 +139,8 @@ public PeekCursor<IndexRow> scan(
             }
         }
 
+        pendingCursors.incrementAndGet();
```

Comment:
Could you please elaborate where do you increment pendingCursros count in case of calling `IndexStorage#get` from withing `PartitionReplicaListener#lookupIndex`?

Reply:
This is an increment for a sorted index; the lookup is incremented for both sorted and hash indexes (AbstractTestIndexStorage#get).

Reply:
Where do you call increment in case of lookup?